### PR TITLE
take advantage of autoattacks against drunks

### PR DESCRIPTION
**Pull request by horrible-little-slime**

### CODE REVIEW

File: src/fights.ts [hunk 1]
```diff
@@ -573,17 +574,25 @@ class FreeRunFight extends FreeFight {
...
-  .abort();
+const pygmyMacro = () =>
+  Macro.if_($monster`pygmy bowler`, Macro.trySkill($skill`Snokebomb`).item($item`Louder Than Bomb`))
+    .if_(
+      $monster`pygmy orderlies`,
+      Macro.trySkill($skill`Feel Hatred`).item($item`divine champagne popper`)
+    )
+    .if_($monster`pygmy janitor`, Macro.item($item`tennis ball`))
+    .if_($monster`time-spinner prank`, Macro.basicCombat())
+    .externalIf(
```
Comment (burningbman) [inline src/fights.ts:585]: I don't understand the reasoning of this additional combat logic, but wanted to make sure it should be included in this change.

Comment (horrible-little-slime) [reply src/fights.ts:585]: I didn't touch the contexts of janitors, bowlers, or orderlies, or even time-pranks--i changed it to a macro-generating function and added the part referring to drunks specificlally

File: src/fights.ts [hunk 2]
```diff
@@ -573,17 +574,25 @@ class FreeRunFight extends FreeFight {
...
+      SourceTerminal.have() || SongBoom.have(),
+      Macro.if_(
```
Comment (phulin) [inline src/fights.ts:587]: do you actually get multiple rounds of stuff to use?

Comment (horrible-little-slime) [reply src/fights.ts:587]: No, but the intent is to only add one of them to the macro. In retrospect I can make this much, much simpler. 1 sec.

**Pull request merged**